.github/workflows/ccpp.yml

```diff
@@ -151,7 +151,7 @@ jobs:
           python3 -m pip -v install --check-build-dependencies --no-build-isolation python/
       - name: Build Python interface documentation
         run: |
-          python3 -m pip install sphinx==5.0.2 # TO REMOVE
+          python3 -m pip install sphinx
```

Comment:
There is still a sphinx pinned at 5.0.2 in the `Dockerfile.test-env` which I think should be removed. Would recommend adding `# TO REMOVE` so this can be cleaned up later.

Reply:
@chrisrichardson this change came from you - can you comment?

Reply:
I tried removing it, and it seems OK.